docs/COMPILE-NIX.md

```diff
@@ -8,24 +8,10 @@ To compile for Microsoft Windows, see [COMPILE-WINDOWS.md](COMPILE-WINDOWS.md)
 
 [0] Dependencies
 
-Unicorn requires few dependent packages as follows.
-
-- For Mac OS X, "pkg-config" and "glib" are needed.
-  Brew users can install "pkg-config" and "glib" with:
-
-        $ brew install pkg-config glib
-
-  NOTE: to build Unicorn universal binaries (which support both 32-bit &
-  64-bit), you need to have glib in universal format. To do that, build &
-  install glib with:
-
-        $ brew install glib --universal
-
-- For Linux, "glib2-dev" is needed.
-  Ubuntu/Debian users can install this with:
-
-        $ sudo apt-get install libglib2.0-dev
+For MacOS, Unicorn requires "pkg-config" package.
```

Comment:
pkgconfig is checked in qemu/configure, but looks like we dont need that anymore. i will remove this dependency.